A/alive2/bundled/patches/cmake-tweaks.patch

```diff
@@ -0,0 +1,24 @@
+diff --git before/alive2/.git/index after/alive2/.git/index
```

Comment:
The wizard should probably not do this. https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/1157